src/pages/DeletedEmails/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import EmailPreview from "../../components/EmailPreview"
+
+function DeletedEmails() {
+    
+    const[deletedEmails, setDeletedEmails] = useState([])
+    
+    function getDeletedEmailData() {
+        fetch("https://email-client-api.dev.io-academy.uk/emails/deleted")
+            .then(res => res.json())
+            .then(deletedEmailData => {
+                // console.log(deletedEmailData.data)
+                setDeletedEmails(deletedEmailData.data)
+            })
+    }
+
+    useEffect (getDeletedEmailData, [])
+    
+    function reverseDate(inputDate) {
+        let splitDate = inputDate.split(" ")[0]
+        let array = splitDate.split("-")
+        let newDate = `${array[2]}-${array[1]}-${array[0]}`
+        return newDate
+    }
```

Comment:
Now that you're using this function a few times, it would be good to move it into it's own file.

Create a new folder in src called `functions` and create a file called reverseDate.js. You'll need to add an `export default reverseDate` at the end just like you have with your components.

Then you can remove all copies of it from your components, but you will have to import it into each component that uses it.